oop/EBook.js

```diff
@@ -0,0 +1,29 @@
+import Book from "./Book.js";
+
+export default class EBook extends Book {
+
+    constructor(title, author, year, fileFormat) {
+        super(title, author, year),  
+        this._fileFormat = fileFormat
+    }
+
+    printInfo() {
+        console.log(`The ebook title is ${this._title}, author is ${this._author}, year of publishing is ${this._year}, file format is ${this._fileFormat}`)
+    }
+
+    get fileFormat() {
+        return this._fileFormat
+    }
+
+    set fileFormat(value) {
+        const allowedFormats = ['PDF', 'TXT', 'HTML']
```

Comment:
As recommendation you can store it as property in class

Reply:
Thanks